src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAnAotApp.cs

```diff
@@ -705,6 +705,34 @@ public void It_publishes_with_implicit_rid_with_NativeAotApp(string targetFramew
             }
         }
 
+        [RequiresMSBuildVersionTheory("17.0.0.32901")]
+        [InlineData(ToolsetInfo.CurrentTargetFramework)]
+        public void It_builds_with_dynamiccodesupport_false(string targetFramework)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                var projectName = "DynamicCodeSupportFalseApp";
+                var testProject = CreateHelloWorldTestProject(targetFramework, projectName, true);
+                testProject.AdditionalProperties["PublishAot"] = "true";
```

Comment:
Did you mean to set DynamicCodeSupport to false instead?

Reply:
No. This test is testing that when `PublishAot=true`, the `System.Runtime.CompilerServices.RuntimeFeature.IsDynamicCodeSupported` feature switch is defaulted to `false`.

Reply:
This way it tests that `PublishAot` implies turning it off, so I think we need this test. We could add another one which sets the property directly, but that one is less important in my mind.

Reply:
> We could add another one which sets the property directly

That's the "KitchenSink" test changes above.